README.md

```diff
@@ -1,22 +1,221 @@
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
+
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/othneildrew/Best-README-Template">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">Resource manager operator</h3>
+
+  <p align="center">
+    Auto-manage  k8s resources
+    <br />
+    <a href="https://github.com/tikalk/resource-manager"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/tikalk/resource-manager">View Demo</a>
+    ·
+    <a href="https://github.com/tikalk/resource-manager/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/tikalk/resource-manager/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgements">Acknowledgements</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
+
+The Resource-Manager operator has built to automatically manage your kubernetes objects.
+
+Here's why:
+* You can easily create schedulded policies that will allow you to manage your kubernetes objects 
+* You shouldn't be doing the same tasks over and over like deleting some pods or namespaces
+
+
+### Built With
+
+* [Operator-SDK](https://sdk.operatorframework.io/)
+* [Go](https://go.dev/)
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+
+### Installation
+
+1. add the helm repo
+   ```bash
+    helm repo add resource-manager https://resourcemanager.github.io/charts/resource-manager-charts   
```

Comment:
waiting for my PR to be merged for creating the following PR which creates this link in github pages.